packages/blade/src/components/BottomSheet/BottomSheet.web.tsx

```diff
@@ -433,6 +433,22 @@ const _BottomSheet = ({
     }
   }, [addBottomSheetToStack, id, isMounted, removeBottomSheetFromStack]);
 
+  // Remove the bottomsheet from the stack, if it's unmounted forcefully
+  React.useEffect(() => {
+    return () => {
+      if (id === undefined) return;
+      removeBottomSheetFromStack(id);
+    };
+  }, [id, removeBottomSheetFromStack]);
+
+  // Disable body scroll lock when the component is unmounted forcefully
+  React.useEffect(() => {
+    const lockTarget = scrollRef.current!;
+    return () => {
+      enableBodyScroll(lockTarget);
```

Comment:
There's API called "enableBodyScroll" that disables body scroll 😅 ??

Reply:
Oh wait just saw the documentation. Did you mean to use `disableBodyScroll`

Reply:
lol i know it's a bit confusing. 

- So when the bottomsheet mounts the body scroll gets locked (ie body scroll gets disabled - disableBodyScroll)
- But when bottomsheet unmounts we want to unlock it (ie enable body to scroll - enableBodyScroll)

Reply:
Ahh `enableBodyScroll` that disables the body scroll "lock" 😆